NinjaResidence/SCENE.cpp

```diff
@@ -48,14 +48,12 @@ void Scene::RunOptionScene(int BGMvolume,int SEvolume) {
 void RevolveZ(CUSTOMVERTEX* Vertex, float Rad, CENTRAL_STATE Central, DWORD  color = 0xffffffff, float tu = 0, float tv = 0, float scaleTu = 1, float scaleTv = 1);
 
 void Scene::LoadAnimation() {
-
 	static int rad = 0;
-	++rad;
+	rad += D3DX_PI;
 	CUSTOMVERTEX closs[4];
 	CENTRAL_STATE CLOSS = { CENTRAL_X,CENTRAL_Y,300,300 };
```

Comment:
今回のレビュー対象では無いですが、
```suggestion
	CENTRAL_STATE CROSS = { CENTRAL_X,CENTRAL_Y,300,300 };
```
では？